### DISCUSSION TITLE
No recovery key, but just want to start over

### DISCUSSION THREAD

**Discussion by irrg** [ente-io/ente#4430] [Q&A]
I'm trying to set up an account and got an error, which is making me think I probably signed up a while back and didn't log my credentials/key. I did a 'forgot password' and I'm fine with blowing up whatever's in my account, but, instead I'm being told to post here. Can someone blow up my account so I can start over?

**Comment by ua741** [comment]
Please write to the support team. After doing some basic validation, they can help you in resolving this.

**Reply by irrg** [reply]
Thanks @ua741 they took care of it. But, the Enti Auth app pushes people here instead. Might be something to change?
